Makefile

```diff
@@ -97,17 +97,15 @@ build_buildx_args=$(BUILDX_EXTRA_ARGS)
 .PHONY: _docker-%
 _docker-%: build-all-platforms
 	docker buildx build --platform $(PLATFORMS) \
-	--tag $(DOCKER_IMAGE_TAG) \
+	--tag $(DOCKER_IMAGE):$(VERSION) \
```

Comment:
I am not sure this is going to work. I think I tried that in the past and also errors with 401, but it is worth another shot.